docs/pages/pack/docs/features/css.mdx

```diff
@@ -52,35 +52,21 @@ Using the CSS `@import` syntax to import other CSS files works **out-of-the-box*
 @import './dark.css';
 ```
 
-<Callout type="info">
-  Turbopack does not currently support absolute URI imports in css, such as `@import "https://example.com/stylesheet.css"`. As a workaround, add
-  `<link rel="stylesheet" href="https://example.com/stylesheet.css" />` to your HTML document. Next.js users using `pages/` should
-  [add this to `pages/_document.js`](https://nextjs.org/docs/messages/no-page-custom-font#possible-ways-to-fix-it).
-</Callout>
-
 ## PostCSS
 
 PostCSS gives you the ability to use plugins to enhance your CSS toolchain. It's been an invaluable tool for integrating libraries like Tailwind and `autoprefixer` into applications.
 
 The most common pattern is adding a `postcss.config.js` file to the root of your application, where you can import and configure your plugins.
 
-We **don't currently offer** the ability to use PostCSS plugins. We may end up adding these out-of-the-box, or make them available via a plugin.
-
-### Workaround for PostCSS
+When Turbopack finds a `postcss.config.js` file, it will automatically process your CSS files with PostCSS in a Node.js worker pool.
```

Comment:
Do we plan to support PostCSS out of the box from Turbopack, or is that a Next.js feature?

Reply:
Sounds like we could do that oob